.gitignore

```diff
@@ -99,7 +99,8 @@ ipython_config.py
 #   This is especially recommended for binary packages to ensure reproducibility, and is more
 #   commonly ignored for libraries.
 #   https://python-poetry.org/docs/basic-usage/#commit-your-poetrylock-file-to-version-control
-#poetry.lock
+poetry.lock
```

Comment:
I use [poetry](https://python-poetry.org/) as my python package manager and I didn't want to include that in this repo